File: C#/Урок 044/Задача 03.cs
class Program
{
    static void Main()
    {
    EnteringPositiveNumber:
        System.Console.Write("Enter a positive number: ");
        int number = System.Convert.ToInt32(System.Console.ReadLine());
        if (number <= 0)
            goto EnteringPositiveNumber;

        bool isPrimeNumber = true;

    CheckingIfNumberIsPrime:
        if (number == 2)
        {
            System.Console.WriteLine("Your number is prime");
            goto EnteringPositiveNumber;
        }

        int divisor = 2;
        if (number % divisor == 0)
            isPrimeNumber = false;

        divisor++;

        if (isPrimeNumber && divisor * divisor < number)
            goto CheckingIfNumberIsPrime;

        if (isPrimeNumber)
            System.Console.WriteLine("Your number is prime");
        else
            System.Console.WriteLine("Your number isn't prime");

        goto EnteringPositiveNumber;
    }
}

File: C++/Урок 027 - Логічні оператори/Задача 03.cpp
#include <iostream>

int main()
{
	std::cout << "Enter your number: ";
	int number;
	std::cin >> number;

	std::cout << "If you see 1 - your number is 1, 2, or 3.\\n";
	std::cout << "If you see 0 - your number is not 1, 2, and 3.\\n";
	std::cout << (number == 1 || number == 2 || number == 3) << std::endl;

	return 0;
}